runtime/src/main/scala/akka/grpc/javadsl/GoogleProtobufSerializer.scala

```diff
@@ -7,13 +7,18 @@ package akka.grpc.javadsl
 import akka.annotation.ApiMayChange
 import akka.grpc.ProtobufSerializer
 import akka.util.ByteString
+import com.google.protobuf.Parser
 
 @ApiMayChange
-class GoogleProtobufSerializer[T <: com.google.protobuf.Message](clazz: Class[T]) extends ProtobufSerializer[T] {
+class GoogleProtobufSerializer[T <: com.google.protobuf.Message](parser: Parser[T]) extends ProtobufSerializer[T] {
+
+  // For binary compatibility in generated sources, can be dropped in version 2.x
```

Comment:
Maybe mark it `@deprecated` so it's easier to find when we can break binary compatibility (which indeed is in 2.x, but also 1.3.x - https://doc.akka.io/docs/akka-grpc/current/binary-compatibility.html#deprecations )

Reply:
Added